src/main/resources/org/jenkins/plugins/lockableresources/LockStep/help-inversePrecedence.html

```diff
@@ -0,0 +1,8 @@
+<div>
+	<p>
+		By default waiting builds get the lock in the same order the requested to acquire it.
+	</p>
+	<p>
+		By checking this option the newest build in the waiting queue will get the lock first.
```

Comment:
What if there are builds of multiple jobs waiting? Including multiple builds of a single job?